CMakeLists.txt

```diff
@@ -60,7 +60,11 @@ endif()
 
 list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
 
-find_package(FairMQ REQUIRED)
+set(CMAKE_THREAD_PREFER_PTHREAD TRUE)
+set(THREADS_PREFER_PTHREAD_FLAG TRUE)
+find_package(Threads REQUIRED)
```

Comment:
Why is this one needed?

Reply:
Ups, missed your question somehow. _Short answer_: Without it would not compile on Fedora 29. _Long answer_: The FairMQ::FairMQ target currently exports all its boost dependencies as public dependencies. One of them seems to require linking against a thread library. So, firstly, we should probably sort out, if we really need to export the boost dependencies as public at all and secondly, if one of them needs a thread library, document it properly. In some systems, one does not see this issue, because the thread support is added by different means.